examples/shadowmap/main.js

```diff
@@ -704,21 +626,54 @@
             this.updateShadowFormat();
             this.updateShadowMapSize();
 
-            this.updateDebugRTT();
+            var i, l, numLights;
+            numLights = ~~this._config.lightNum;
```

Comment:
not a fan of this, If you fixed the content with the url parser in Example I would prefer we remove this kind of tricks